content/tutorials/native-aa-multisig/10.index.md

```diff
@@ -41,31 +43,48 @@ Atlas is a smart contract IDE that lets you write, deploy, and interact with con
 
 1. Navigate into the project directory:
 
+    :test-action{actionId="move-into-project"}
+
     ```sh
     cd custom-aa-tutorial
     ```
 
 1. For the purposes of this tutorial, we don't need the example contracts related files.
   So, proceed by removing all the files inside the `/contracts` and `/deploy` folders manually or by running the following commands:
 
+    :test-action{actionId="delete-templates"}
+
     ```sh
     rm -rf ./contracts/*
     rm -rf ./deploy/*
     ```
 
 1. Add the ZKsync and OpenZeppelin contract libraries:
 
-    ```sh
-    yarn add -D @matterlabs/zksync-contracts @openzeppelin/contracts@4.9.5
+    :test-action{actionId="install-deps"}
+
+    ::code-group
+
+    ```shell [npm]
+    npm install -D @matterlabs/zksync-contracts @openzeppelin/contracts@4.9.5 @matterlabs/hardhat-zksync-deploy@1.3.0
```

Comment:
we shouldnt be installing old deps and should make use of `@matterlabs/hardhat-zksync`